spec/features/consumer/caching/darkwarm_caching_spec.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+feature "Darkswarm data caching", js: true, caching: true do
+  let!(:taxon) { create(:taxon, name: "Cached Taxon") }
+  let!(:property) { create(:property, presentation: "Cached Property") }
+
+  let!(:producer) { create(:supplier_enterprise) }
+  let!(:distributor) { create(:distributor_enterprise, with_payment_and_shipping: true, is_primary_producer: true) }
```

Comment:
I wonder why rubocop is not complaining about line too long here

Reply:
very likely that this file is whitelisted in rubocop config.

Reply:
Is it a typo in the file name as well here? darkwarm -> darkswarm?
If this file is added as part of this PR maybe it could be updated?